### DISCUSSION TITLE
Metrics Exporting

### DISCUSSION THREAD

**Discussion by SaidineshBijjam** [open-telemetry/opentelemetry-cpp-contrib#464] [Q&A]
When using OpenTelemetry instrumentation in Java, I am able to export both metrics and traces. However, with the native OpenTelemetry modules for Apache/Nginx, I have observed that only traces can be exported, and metrics support is currently missing. Although I understand that metrics can be exported using OpenTelemetry collectors, I am curious if there are plans to support direct metrics export via the native modules in the future or if it is already available ?

**Comment by lalitb** [comment]
@DebajitDas  can you reply here. Thanks. @svrnm

**Comment by DebajitDas** [comment]
@SaidineshBijjam 
Yes, currently metrics are not exported from Otel-webserver-module. 
Do you have some use-case or what kind of metrics that you are expecting from otel webserver module. 
I am aware of server metrics which can be collected from collector. Are you talking about the same ?

CC @aryanishan1001
